front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts

```diff
@@ -93,10 +93,9 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
     });
 
     if (this.transaction?.transaction_type_identifier) {
-      // Remove transaction_id from list of validate properties because it will be added in the back end.
-      const fieldsToValidate: string[] = this.validateService
-        .getSchemaProperties(this.schema)
-        .filter((p) => p !== 'transaction_id');
+      let fieldsToValidate: string[] = this.getFieldsToValidate();
+      // Remove properties populated in the back-end from list of properties to validate
+      fieldsToValidate = fieldsToValidate.filter((p) => p !== 'transaction_id' && p !== 'donor_committee_name');
```

Comment:
it seems like we'd have to do this after every time we call `getFieldsToValidate()`.  If that's the case, should we move it inside that function?

Reply:
I had created that function to give more control to any inherited transaction type component more control over the validation fields but eventually did not need it on how the refactoring turned out. I can roll back that additional method.